src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -124,6 +126,28 @@ def get(self, request, organization):
                 elif key == "role":
                     queryset = queryset.filter(role__in=value)
 
+                elif key == "isInvited":
+                    isInvited = "true" in value
```

Comment:
Should 'isInvited:truefools' be a valid parameter?

Reply:
no, `value` is an array. Since you can do `isInvited:true isInvited:false` (which in this case will evaluate to true)